docs/source/miscellaneous/dropwizard-metrics.rst

```diff
@@ -0,0 +1,57 @@
+the Metrics `Getting Started Guide <http://metrics.dropwizard.io/3.1.0/getting-started/#>`__.
+
+**Gauges**
+
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.numBlacklistedHosts``
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.<host>.requestConnectionExceptionProportion``
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.<host>.requestFailureProportion``
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.requestConnectionExceptionProportion``
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.requestFailureProportion``
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPoolingContainer.<host>.meanActiveTimeMillis``
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPoolingContainer.<host>.meanBorrowWaitTimeMillis``
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPoolingContainer.<host>.meanIdleTimeMillis``
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPoolingContainer.<host>.proportionDestroyedByBorrower``
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPoolingContainer.<host>.proportionDestroyedByEvictor``
+
+**Histograms**
+
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.byteSizeTx``
+
+**Meters**
+
+- ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.<host>.requestConnectionExceptions``
+- ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.<host>.requestExceptions``
+- ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.<host>.requests``
+- ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.requestConnectionExceptions``
+- ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.requestExceptions``
+- ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.requests``
+- ``com.palantir.atlasdb.transaction.api.LockAwareTransactionManager.runTaskReadOnly.failures``
+- ``com.palantir.atlasdb.transaction.api.LockAwareTransactionManager.runTaskWithRetry.failures``
+
+Additional failure counts will be dynamically generated based on the returned exceptions, so you may see metrics like
+the following:
+
+- ``com.palantir.atlasdb.transaction.api.LockAwareTransactionManager.runTaskWithRetry.failures.java.lang.IllegalStateException``
+
+**Timers**
+
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.commitAcquireLocks``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.commitCheckingForConflicts``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.commitPutCommitTs``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.commitTotalTimeSinceTxCreation``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.commitWrite``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.getRows``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.millisForPunch``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.processedRangeMillis``
```

Comment:
So I'm pretty sure these times are all actually still reported. `metricRegistry.timer` automatically registers a metric if none yet exists, and these calls still exist in `SnapshotTransaction.java`.